infra/terraform/modules/service/batch.tf

```diff
@@ -27,6 +27,22 @@ locals {
 
       image = "${var.batch.repository}:${var.batch.version}"
 
+      environment = [
+        {
+          name  = "ENVIRONMENT_NAME"
+          value = var.environment
+        },
+        {
+          name  = "APP_VERSION"
+          value = var.batch.version
+        },
+      ],
+
+      runtimePlatform = {
+        operatingSystemFamily = "LINUX",
+        cpuArchitecture       = "ARM64"
```

Comment:
Long term we probably want to make this architecture agnostic right? Makes sense that we might not do this for now given the time constraints but as part of making it truly portable as per good practice might be something we consider in near future.

Reply:
No, I'm not sure why we would make it agnostic. We don't need the application to be portable across different architectures. If we want an `x86_64` build then we can easily do that by changing the various `arm64` references and away we go.

`arm64` was chosen because it is what developers and platform engineers are using locally.

Reply:
The reason we might is that it restricts us from a cloud perspective. As an example we cannot use spot instances for batch (not to say that AWS won't release this feature in the future for x64 architectures). this aligns to the principle of trying to optimise portability generally.

At this stage i agree there is limited benefits of doing this that we can prove (other than the usage of spot which at this stage we have not proven the benefit) especially given most are using m1 macs. You can emulate this using docker desktop in any case.

Reply:
Sorry, I mean that this doesn't need to be portable as the move to `x86_64` is a few lines change. To make this portable would not be worth the work.